utils/openocd/Makefile

```diff
@@ -8,14 +8,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=openocd
-PKG_SOURCE_VERSION:=v0.11.0-rc1
+PKG_SOURCE_VERSION:=v0.11.0
 PKG_VERSION:=$(PKG_SOURCE_VERSION)
 PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
 PKG_SOURCE_URL:=git://git.code.sf.net/p/openocd/code
```

Comment:
Do you insist? I would prefer git so that it would be easy to package a reasonably tested but not officially released version in the future. What is the advantage of using https here?

Reply:
Just checked. Looks like you can just do https://git.code.sf.net/p/openocd/code . It's used elsewhere in the tree.

The benefit is that https goes through some corporate firewalls. Note that I'm not asking to use a release tarball.

Reply:
Got it, thank you!